token-syncer/test/token_syncer/waiter_test.clj

```diff
@@ -90,7 +90,7 @@
       (let [error (Exception. "exception from test")]
         (with-redefs [make-http-request (fn [in-http-client-wrapper in-endopint-url & in-options]
                                           (is (= http-client-wrapper in-http-client-wrapper))
-                                          (is (= (str test-cluster-url "/token")) in-endopint-url)
+                                          (is (= (str test-cluster-url "/tokens") in-endopint-url))
```

Comment:
Is this related to this change?

Reply:
Oops sorry, this is a bug fix. The assertions were previously ignoring the `in-endopint-url` and treating it as a the message for a failed assertion.